config/jobs/kubernetes/sig-storage/sig-storage-kind.yaml

```
@@ -54,6 +54,54 @@ presubmits:
             cpu: "2"
             memory: "6Gi"
 
+  # This jobs runs e2e.test with a focus on tests for the VolumeAttributesClass feature (currently alpha)
+  # on a kind cluster
+  - name: pull-kubernetes-e2e-storage-kind-vac-feature
+    always_run: false
+    optional: true
+    decorate: true
+    path_alias: k8s.io/kubernetes
+    cluster: eks-prow-build-cluster
+    skip_branches:
+    - release-\d+\.\d+
+    annotations:
+      testgrid-dashboards: presubmits-kubernetes-nonblocking
+      testgrid-tab-name: pull-kubernetes-e2e-storage-kind-vac-feature
+      testgrid-alert-email: kubernetes-sig-storage-test-failures@googlegroups.com
+      description: Run storage tests that need the VolumeAttributesClass feature in a KIND cluster.
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/krte:v20240111-cf1d81388e-master
+        command:
+        - wrapper.sh
+        - bash
+        - -c
+        - curl -sSL https://kind.sigs.k8s.io/dl/latest/linux-amd64.tgz | tar xvfz - -C "${PATH%%:*}/" && e2e-k8s.sh
+        env:
+        # EventedPLEG is disabled temporarily for https://github.com/kubernetes/kubernetes/issues/122721
+        - name: FEATURE_GATES
+          value: '{"VolumeAttributesClass":true, "EventedPLEG":false}'
+        - name: RUNTIME_CONFIG
+          value: '{"api/alpha":"true", "api/ga":"true"}'
+        - name: FOCUS
+          value: VolumeAttributesClass
```

More specific perhaps?

```suggestion
          value: \[Feature:VolumeAttributesClass\]
```